Wikipedia/Code/ImageController.swift

```diff
@@ -32,27 +32,6 @@ let WMFExtendedFileAttributeNameMIMEType = "org.wikimedia.MIMEType"
 open class ImageController : NSObject {
```

Comment:
I think this class is only used in a small unit test class, so maybe we can delete it entirely.